1-exercises/I-string-replace/exercise.js

```diff
@@ -13,7 +13,12 @@
 let story =
   "I like dogs. One day I went to the park and I saw 10 dogs. It was a great day.";
 
-let result = story.replace("", "");
+let result1 = story.replace('great', "brilliant");
+let result2 = result1.replace(10,100000);
+let result3 = result2.replace(/day/gi,'night');
```

Comment:
This could have worked even if it was a regular expression and it would probably be easier. For example:

`let result3 = result2.replace('day','night');`